docker/Makefile

```diff
@@ -13,6 +13,13 @@ endif
 build: .TEST
 	docker build --network host --build-arg=EMSCRIPTEN_VERSION=${version} --tag emscripten/emsdk:${version} .
 
+test: .TEST
+	# test as non-root
+	# FIXME: the `-e HOME=/tmp` is a workaround for:
+	# https://github.com/emscripten-core/emsdk/issues/535
+	docker run --rm -u `id -u`:`id -g` -e HOME=/tmp --net=host emscripten/emsdk:${version} \
+		/bin/bash -c "`cat test_dockerimage.sh`"
```

Comment:
This construction seems a little bit hacky... why not just leave test_dockerimage.sh inside the container?

Reply:
Sorry, I was overthinking the issue. We could indeed just run `/emsdk/docker/test_dockerimage.sh` instead.

Commit https://github.com/emscripten-core/emsdk/pull/571/commits/118f0919659c43ab7c282c95917ccdf8bf952fc8 removes this hackish solution.